protostar-rust/src/cheatcodes_hint_processor.rs

```diff
@@ -195,19 +292,18 @@ fn execute_cheatcode_hint(
             };
             let account_tx = AccountTransaction::Declare(tx);
             let block_context = &BlockContext::create_for_account_testing();
-
-            let actual_execution_info = account_tx
+            let tx_result = account_tx
                 .execute(blockifier_state, block_context)
                 .expect("Failed to execute declare transaction");
 
-            let class_hash_int =
-                i64::from_str_radix(&class_hash.to_string().replace("0x", "")[..], 16)
-                    .expect("Failed to convert hex string to int");
-
-            insert_value_to_cellref!(vm, result, Felt252::from(class_hash_int))?;
+            insert_value_to_cellref!(
+                vm,
+                result,
+                Felt252::from_str_radix(&class_hash.to_string().replace("0x", "")[..], 16).unwrap()
```

Comment:
Can we have a utility function for that?

Reply:
You mean for converting 0x prefixed strings to Felt252 or specifically the `ClassHash` type.

Reply:
Whichever one you feel like would be more useful in the future